### DISCUSSION TITLE
Way to swap Ctrl and Caps Lock key on niri?

### DISCUSSION THREAD

**Discussion by thepragandsensdiary** [YaLTeR/niri#705] [Q&A] [answered]
Someone could find a cute and tidy way to swap the ctrl and caps lock keys on niri?
On GNOME I achieve that using this `gsettings` command:
`gsettings set org.gnome.desktop.input-sources xkb-options "['caps:ctrl_modifier', 'ctrl:swapcaps']"`
And it just works as one [chess nerd](https://www.youtube.com/watch?v=z0v8DBQB3uQ) wisely said.

Could it be achieved with just a couple of config.kdl lines?

Pwease hewp daddy yalter

**Comment by YaLTeR** [comment, marked as answer]
Yes, put it into niri xkb options: https://github.com/YaLTeR/niri/wiki/Configuration:-Input#layout

`options "caps:ctrl_modifier,ctrl:swapcaps"`

**Reply by thepragandsensdiary** [reply]
Tnks for educate this uneducated man. Nice project YaLTeR <3
